cpp/src/rolling/rolling.cu

```diff
@@ -58,7 +58,7 @@ template <typename InputType,
           typename agg_op,
           aggregation::Kind op,
           bool has_nulls>
-std::enable_if_t<op == aggregation::COUNT_VALID || op == aggregation::COUNT_ALL, bool> __device__
+std::enable_if_t<op == aggregation::COUNT_VALID, bool> __device__
```

Comment:
Is there a reason to use return-type-SFINAE over template-parameter-SFINAE?

Reply:
No, sir, except to retain the erstwhile  convention for `process_rolling_window()`. I would prefer the template parameter, myself.
Would you recommend changing this out?

Reply:
If it's not a big deal. We use template parameter SFINAE almost everywhere.

Reply:
> If it's not a big deal.

Hardly any trouble. I've switched this to template-parameter based SFINAE.